code/modules/mob/living/silicon/silicon_defense.dm

```diff
@@ -61,11 +61,38 @@
 			M.visible_message("<span class='notice'>[M] pets [src]!</span>", \
 							"<span class='notice'>You pet [src]!</span>")
 			playsound(loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
-		if("grab")
+		if(INTENT_GRAB)
 			grabbedby(M)
+		if(INTENT_DISARM)
+			tip_over(M)
 		else
 			M.do_attack_animation(src, ATTACK_EFFECT_PUNCH)
 			playsound(loc, 'sound/effects/bang.ogg', 10, 1)
 			visible_message("<span class='notice'>[M] punches [src], but doesn't leave a dent.</span>", \
 						"<span class='notice'>[M] punches [src], but doesn't leave a dent.</span>")
 	return FALSE
+
+/*
+Rotates a silicon's sprite 180 degrees, returns true if successful, false otherwise. All of the un-tipping is handled by the tilted component
+*/
+/mob/living/silicon/proc/tip_over(mob/living/carbon/human/user)
+	if(is_being_tipped)
+		return
+	visible_message(
+		"<span class='notice'>[user] starts to tip [src] over!</span>",
+		"<span class='danger'>[user] starts to tip you over!</span>")
+	is_being_tipped = TRUE
+	if(do_mob(user, src, tipping_time))
+		var/tip_turf = get_step(src, get_dir(user, src))
+		if(!fall_and_crush(tip_turf, tilt_damage, weaken_time = 0, knockdown_time = 0, angle = HALF_TURN, ignore_gravity = TRUE, rightable = TRUE, untilt_time = tipping_time))
+			is_being_tipped = FALSE
+			return FALSE
+		drop_hat()
+		visible_message(
+			"<span class='notice'>[user] tips [src] over!</span>",
+			"<span class='danger'>[user] tips you over!</span>",
+			"<span class=warning'>You the clatter of metal on the ground.</span>")
```

Comment:
```suggestion
			"<span class='warning'>You the clatter of metal on the ground.</span>")
```

Reply:
Is this meant to be "you hear the clatter" or something?